vraptor-core/src/main/java/br/com/caelum/vraptor/serialization/gson/DateGsonConverter.java

```diff
@@ -61,5 +61,9 @@ public Date deserialize(JsonElement json, Type typeOfT, JsonDeserializationConte
 			throw new JsonSyntaxException(json.getAsString(), e);
 		}
 	}
+	
+	public String getPattern(){
```

Comment:
I'd rename to `getDateFormat` and returns a `SimpleDateFormat` instead a String to keep symmetry with basic converters.

```java
protected DateFormat getDateFormat() 
    new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
}
```